sensors/omron_d6t/omron_d6t.py

```diff
@@ -0,0 +1,96 @@
+import time, crcmod.predefined
+from sensors.sensor import Sensor
+
+class OmronD6T(Sensor):
+    __START_COMMAND = 0x4c
+    __ARRAY_SIZE = 16
+    __BUFFER_LENGTH = 35
+    __PEC_BUFFER = [0x15]
+    __MAX_CELLS_ABOVE_ROOM_TEMPERATURE = 10
```

Comment:
Out of curiosity, why make the constants private?

Reply:
Wasn't sure if it was the more pythonic way since i've seen it in our code. I can totally change it though.

Reply:
I don't know the answer. Not a priority, and wouldn't be a pain to change later.

Reply:
I don't think either way is more pythonic. 

It would not be necessary to expose these constants publicly, as these are more or less just constants that belong to this sensor only, and wouldn't need to be accessed outside of this module.